modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -327,6 +329,160 @@ const Quirks = (editor: Editor): Quirks => {
     });
   };
 
+  // This helper function, deletes the content created by Chrome which has extra font-family style and replaces
+  // it with the original content saved on keydown which does not have font-family
+  const removeExtraFontFamilyOnKeyup = (editor: Editor, specialDelete: Cell<boolean>, content: Cell<Optional<string>>) => {
+    editor.on('keyup', (e) => {
+      if (isDefaultPrevented(e) || e.key !== 'Backspace' && e.key !== 'Delete' || !specialDelete.get()) {
+        return;
+      }
+
+      const rng = selection.getRng();
+      const container = rng.startContainer;
+      const root = dom.getRoot();
+      const parent = Optional.from(Traverse.parents(SugarElement.fromDom(container)).find((node) => node.dom.nodeName.toLowerCase() === 'li'));
```

Comment:
You can use PredicateFind.ancestor or PredicateFind.closest here also this should likely be constrained to the root so it doesn't try to find parents outside the root element in inline mode.

Reply:
Done!